packages/next/src/server/app-render/dynamic-rendering.ts

```diff
@@ -40,14 +40,15 @@ export function markCurrentScopeAsDynamic(
   store: StaticGenerationStore,
   expression: string
 ): void {
+  const pathname = getPathname(store.urlPathname)
```

Comment:
Is this the same as `store.pagePath`? (I can never remember what that is)

Reply:
pagePath is the page route, for app router it's `/pathname/page` for pages and `/pathname/route` for custom routes, and the type was string | undefined. So url path is pretty safe to go that I only need to remove the rsc query